capi_problems.rst

```diff
@@ -80,6 +80,11 @@ describes this complex state of affairs in terms of the
 actions that different stakeholders need to perform through
 the C API.
 
+There are two main groups of the users of the C API:
+
+* External users. They only consume the API.
+* CPython developers. They define, implement, and consume the API inside the CPython code.
```

Comment:
In terms of the structure of this section, I think this binary division into "us" and "them" gives too much weight to the internal usage in cpython. I tried in the section to include a list of types of users, and this lumps everything other than cpython core into one category, with the others being sub-categories.

Can we, instead of talking about "two main types of users", just add the internal usage in cpython core as one of the stakeholders, and in the cpython core section talk about what they do or don't need?